### ISSUE TITLE
ImportError: cannot import name '_is_openai_v1'

### ISSUE THREAD

**Issue by kbatbouta** [langchain-ai/langchain#15671]
### System Info

Hi, I am encountering this error when trying to import anything from the `langchain.embeddings` on Amazon linux AMI with python 3.9 and `langchain==0.0.350`
```python
Traceback (most recent call last):
  File "/home/ec2-user/app/search/./app.py", line 9, in <module>
    from search import make_chain, postprocess
  File "/home/ec2-user/app/search/search.py", line 6, in <module>
    from langchain.embeddings import HuggingFaceInstructEmbeddings
  File "/home/ec2-user/.local/lib/python3.9/site-packages/langchain/embeddings/__init__.py", line 62, in <module>
    from langchain.embeddings.openai import OpenAIEmbeddings
  File "/home/ec2-user/.local/lib/python3.9/site-packages/langchain/embeddings/openai.py", line 1, in <module>
    from langchain_community.embeddings.openai import (
ImportError: cannot import name '_is_openai_v1' from 'langchain_community.embeddings.openai' (/home/ec2-user/.local/lib/python3.9/site-packages/langchain_community/embeddings/openai.py)
```
The error currently occurs when calling 
```python
from langchain.embeddings import HuggingFaceInstructEmbeddings
```

My requirements.txt file looks like this:
```
fastapi==0.105.0
lancedb==0.3.4
langchain==0.0.350
langserve==0.0.36
numpy==1.26.2
pandas==2.1.4
Requests==2.31.0
uvicorn==0.24.0.post1
```

I should note that I've tried reinstalling langchain, openai and transfomers. I've also tried python 3.10 and got the same error. 

I should also note that none of my modules are called openai.

### Information

- [ ] The official example notebooks/scripts
- [X] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [X] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

Call `from langchain.embeddings import HuggingFaceInstructEmbeddings`  or any of the embeddings modules

### Expected behavior

Should be able to import without errors.

**Comment by JungeAlexander** [comment]
@kbatbouta just had the same issue. Updating to langchain 0.1.0 (from 0.0.350, just like you), solved the problem for me.

**Comment by hwchase17** [comment]
apologies - we had been importing a private method `_is_openai_v1` in older versions of langchain. as @JungeAlexander noted, the fix is to upgrade your langchain package (0.0.350 -> 0.1.0 should be pretty easy)

sorry for any confusion!

I would also recomend importing directly from langchain_community.embeddings from now on
